components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemBuy.vue

```diff
@@ -13,11 +13,12 @@
         </div>
 
         <NeoButton
-          class="button-height no-border-left"
+          class="button-height square-button-width no-border-left"
           data-testid="item-add-to-cart"
           @click="onClickShoppingCart">
           <NeoIcon
             size="medium"
+            class="is-flex is-align-items-center is-justify-content-center"
```

Comment:
I think you can use `class="icon"` instead

Reply:
Updated